.husky/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
I ran `shellcheck` on the Husky pre-commit file, and its complaining:

```
shellcheck .husky/pre-commit

In .husky/pre-commit line 2:
. "$(dirname "$0")/_/husky.sh"
  ^--------------------------^ SC1090: Can't follow non-constant source. Use a directive to specify location.

For more information:
  https://www.shellcheck.net/wiki/SC1090 -- Can't follow non-constant source....
make: *** [Makefile:9: shellcheck] Error 1
```

Do you have any ideas on how to fix this properly? We could just put a ignore directive in, but that's not fixing the root problem, that only tells `shellcheck` to stop bothering us...

Reply:
```suggestion
# shellcheck source=/dev/null
. "$(dirname "$0")/_/husky.sh"
```